### ISSUE TITLE
Cannot tell when Yasmine server/listener is up and running from startup logging.

### ISSUE THREAD

**Issue by rcasey-earthscope** [iris-edu/yasmine-stationxml-editor#46]
When starting up Yasmine in a text terminal, the user is presented with a long period of scrolling output text as the system starts up.  The start up is slow, yes, but the real issue is that there is a point where the scrolling stops, and we are left at the cryptic state of:

yasmine-frontend  | [LOG] Fashion build complete for /code/frontend/build/temp/development/yasmine/sass/yasmine-all.scss
yasmine-frontend  | [LOG] Fashion build completed in 7.488 sec.
yasmine-frontend  | [INF] Appending content to /code/frontend/bootstrap.js
yasmine-frontend  | [INF] Writing content to /code/frontend/classic.json
yasmine-frontend  | [INF] Loading Build Environment
yasmine-frontend  | [INF] Waiting for changes...
yasmine-frontend  | [WRN] Use of 'require()' will be deprecated: /code/frontend/ext/classic/theme-neutral/sass/etc/mixins/font-icon.scss:1
yasmine-frontend  | [WRN] Use @import instead
yasmine-frontend  | [LOG] Fashion waiting for changes...

The thing is, it's hard to tell if the initialization has actually finished.  In fact, it hasn't.  Waiting for about two minutes yields another process, that of loading the NRL, that suddenly logs to the screen.

yasmine-backend   | INFO:yasmine.app.helpers.nrl.nrl_helper:Unzipping NRL
yasmine-backend   | INFO:yasmine.app.helpers.nrl.nrl_helper:NRL has been unzipped
yasmine-backend   | INFO:yasmine.app.helpers.nrl.nrl_helper:Creating an NRL key files

Is Yasmine ready for use?  I have been waiting 4 minutes to start using it, and there is nothing that says that Yasmine is ready.  My only clue is that logging to the screen seems to stop, which is kind of a crummy feature.

**Comment by gillian-earthscope** [comment]
Yes, that's very user unfriendly and the source of much frustration. The
final message should read:
yasmine-backend   | INFO:yasmine.app.helpers.nrl.nrl_helper:NRL key files
have been created

-Gillian
